app/src/main/java/com/framgia/fsalon/screen/scheduler/SchedulerFragment.java

```diff
@@ -31,7 +33,8 @@ public void onCreate(Bundle savedInstanceState) {
         SchedulerContract.Presenter presenter =
             new SchedulerPresenter(mViewModel, new ManageBookingRepository(
                 new ManageBookingRemoteDataSource(
-                    FSalonServiceClient.getInstance())));
+                    FSalonServiceClient.getInstance())), new SalonRepository(new
+                SalonRemoteDataSource(FSalonServiceClient.getInstance())));
```

Comment:
ERROR: 'operator new' child have incorrect indentation level 16, expected level should be 20.